recipes/libwebp/all/conanfile.py

```diff
@@ -33,24 +33,17 @@ class LibwebpConan(ConanFile):
     }
 
     def export_sources(self):
-        for p in self.conan_data.get("patches", {}).get(self.version, []):
```

Comment:
Btw, is this still needed in conan v2? Doesn't it provide a default implementation?

Reply:
Yes, still required: https://docs.conan.io/en/2.0/reference/conanfile/attributes.html#exports-sources

And it will be required always because Conan 2.0 adopted even a more explicit flow and in Conan 1.x.